usecases/schema/class.go

```diff
@@ -330,104 +336,121 @@ func (h *Handler) migrateClassSettings(class *models.Class) {
 	}
 }
 
-func migratePropertySettings(prop *models.Property) {
-	migratePropertyDataTypeAndTokenization(prop)
-	migratePropertyIndexInverted(prop)
+func migratePropertySettings(props ...*models.Property) {
+	migratePropertyDataTypeAndTokenization(props...)
+	migratePropertyIndexInverted(props...)
+}
+
+func mergeClassExistedProp(class *models.Class, props ...*models.Property) {
+	existingNames := map[string]int{}
```

Comment:
same here, you can use `map[string]struct{}` to be even more memory efficient

Reply:
same as above